### PR TITLE
[ALLUXIO-3100] Avoid spurious errors when TTL hits deleted files

### PR DESCRIPTION
**Pull request by aaudiber**
https://alluxio.atlassian.net/browse/ALLUXIO-3100

### CODE REVIEW

**Comment by aaudiber** [comment]
Jenkins, test this please

File: core/server/master/src/main/java/alluxio/master/file/InodeTtlChecker.java [hunk 1]
```diff
@@ -60,6 +61,9 @@ public void heartbeat() {
         try (LockedInodePath inodePath = mInodeTree
             .lockFullInodePath(inode.getId(), InodeTree.LockMode.READ)) {
           path = inodePath.getUri();
+        } catch (FileDoesNotExistException e) {
+          // The inode has already been deleted, nothing needs to be done.
+          return;
```
Comment (calvinjia) [inline core/server/master/src/main/java/alluxio/master/file/InodeTtlChecker.java]: should this be `continue`?

Comment (aaudiber) [reply core/server/master/src/main/java/alluxio/master/file/InodeTtlChecker.java]: updated, thanks

**Comment by yupeng9** [comment]
@aaudiber do you think we need a test to cover this case?

**Comment by aaudiber** [comment]
@yupeng9 test added

**Pull request merged**